sql/moz-fx-data-shared-prod/search_terms_derived/sanitation_job_metadata/metadata.yaml

```diff
@@ -0,0 +1,31 @@
+---
+friendly_name: Sanitized Firefox Search Terms
+owners:
+  - ctroy@mozilla.com
+labels:
```

Comment:
I think this is accurate but I'm not sure if this is specifically used by other systems so we may want to remove it.

Reply:
I don't think this is actually used to cause any downstream behavior. The `incremental` label is documentation only.